### ISSUE TITLE
Unnecessary comma when dropping a worn object?

### ISSUE THREAD

**Issue by eriktorbjorn** [the-infocom-files/planetfall#7]
```
>DROP UNIFORM
You'll have to take it off, first.
```

Should there really be a comma between "off" and "first"? It looks strange to me...

**Comment by eriktorbjorn** [comment]
In Stationfall, the corresponding message is "You'll have to remove the Patrol uniform first." without a comma.

**Comment by eriktorbjorn** [comment]
Looking through the ZIL source code for all the games currently in The Infocom Files, I found the following:

- Planetfall, ```TAKE-IT-OFF```, "You'll have to take it off, first."
- Planetfall, ```SAFETY-WEB-F```, "You'll have to stand up, first."
- Planetfall, ```BED-F```, "You'll have to stand up, first."
- Planetfall Solid Gold, same as regular Planetfall.
- Sorcerer, ```WAXER-F```, "Better put it down, first."

There was one in A Mind Forever Voyaging as well, but it was commented out. So it looks like it's in some of Steve Meretzky's early games, but not in his later ones?

**Comment by Kweepa** [comment]
This is perfectly legitimate English.

**Comment by eriktorbjorn** [comment]
Fair enough. It just looked a bit strange to me, as if it was saying "You have to take it off [pause] first."

**Comment by AlexProudfoot** [comment]
It seems strange enough to me to warrant discussion with someone who has an english degree. I’ll get back to you with the result.

**Comment by AlexProudfoot** [comment]
If dramatic emphasis makes no sense, I think we should lose the comma.

**Comment by Kweepa** [comment]
I don't think it's there for dramatic emphasis, just to make the message more colloquial. I wouldn't edit the text in this case.
